INSTALL.md

```diff
@@ -1,14 +1,27 @@
 # Dependencies
 
+## Required
+
 Before you can start compiling tinc from a fresh git clone, you have
 to install the very latest versions of the following packages:
 
-- `meson`
-- `ninja`
+- `meson` or `muon` (read below)
+- `ninja` or `samurai`
```

Comment:
Maybe we should also test building with samurai?

Reply:
It's already used on alpine (where it's aliased to ninja), but I'll add to to the muon job anyway.